src/components/Footer.jsx

```diff
@@ -1,5 +1,30 @@
+import { AiFillInstagram } from "react-icons/ai";
+import { IoLogoFacebook } from "react-icons/io";
+import { FaTiktok } from "react-icons/fa";
+
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="w-full h-56 bg-rishi-green absolute inset-x-0 bottom-0">
```

Comment:
change h-56 to pt-16. it'll make styling the footer on smaller screens easier

Reply:
what is the difference of using this?